modules/manage/pages/manage-settings/general-settings.adoc

```diff
@@ -192,17 +194,32 @@ The top section of the panel appears as follows:
 
 image::manage-settings/query-settings-top.png["The top half of the Query Settings panel",548,align=center]
 
-Specify either *Unrestricted* or *Restricted*, to determine which URLs are permitted to be accessed by the `curl` function.
-If *Unrestricted* (the default) is specified, all URLs can be accessed.
-If *Restricted* is specified, the UI expands, to display configurable fields into which the URLs allowed and disallowed can be entered.
+Under *CURL() Function Access*, specify either *Unrestricted* or *Restricted*, to determine which URLs can be accessed by the `curl` function.
+
+* If *Unrestricted* (the default) is specified, all URLs can be accessed.
```

Comment:
Standards nitpick: can we make this and the following bullet point active voice? ("When you specify...")

Reply:
Done